flink-ai-flow/ai_flow_plugins/scheduler_plugins/airflow/airflow_scheduler.py

```diff
@@ -32,9 +35,10 @@
 from airflow.utils.db import create_session
 from airflow.utils.state import State
 from airflow.contrib.jobs.scheduler_client import EventSchedulerClient, SCHEDULER_NAMESPACE, ExecutionContext
+from airflow.utils.dates import parse_execution_date
 
 
-class AirFlowScheduler(Scheduler):
+class AirFlowSchedulerBase(Scheduler, ABC):
```

Comment:
Why does the `AirFlowSchedulerBase` inherited from `ABC`? There is no any abstract methods in `AirFlowSchedulerBase`.